extensions/kubernetes-config/runtime/src/main/java/io/quarkus/kubernetes/client/runtime/KubernetesConfigSourceConfig.java

```diff
@@ -28,4 +28,10 @@
     @ConfigItem
     public Optional<List<String>> configMaps;
 
+    /**
+     * Namespace to look for config maps
+     */
+    @ConfigItem
+    public Optional<String> namespace;
```

Comment:
I am generally not super happy about having every configMap use the same namespace, but when I was writing the initial version of the extension I couldn't come up with an easy enough to use configuration that would cater both no namespace and different namespace configMaps, so I guess we can live with this.